spec/system/consumer/white_label_spec.rb

```diff
@@ -157,6 +157,36 @@
 
       it_behaves_like "does not hide the OFN navigation"
     end
+
+    context "manage hide_groups_tab preference when distributor belongs to a group" do
```

Comment:
I would think this spec belongs with other tests relating to accessing groups.

But.. I just searched and can't find any system spec about being able to access the groups tab in the first place. Hmm I feel a new issue coming on..